src/pages/access-recovery-confirm/lib/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './map-errors';
+export * from './validate-password';
```

Comment:
Password validators can be placed inside `src/lib/password`. Because it can be used on settings page, on register page, on access recovery page.